testscripts/test3.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by TimH on 28-Mar-17.
```

Comment:
Those are only scripts for testing purposes. They get removed once everything works